.github/workflows/linuxjs-tests.yml

```diff
@@ -13,10 +13,7 @@ jobs:
     name: JS
     strategy:
       matrix:
-        # Fix for https://github.com/jupyterlab/jupyterlab/issues/13903
-        include:
-          - group: js-debugger
-            python-version: '3.11'
```

Comment:
Looks like this should also fix https://github.com/jupyterlab/jupyterlab/issues/13946